src/openvdb/OpenVDBDecoder.cpp

```diff
@@ -404,22 +387,29 @@ namespace Zibra::OpenVDBSupport
             sparseFrame.perChannelGridInfo[channelIndex].statistics.voxelCount =
                 perChannelBlockCount[channelIndex] * ZIB_BLOCK_ELEMENT_COUNT;
 
-            const double volume = originGrid->constTransform().voxelVolume();
+            const auto& gridTransform = orderedChannels[channelIndex].second->constTransform();
+            const double volume = gridTransform.voxelVolume();
             if (!MathHelpers::IsNearlyEqual(volume, 0.))
             {
-                sparseFrame.perChannelGridInfo[channelIndex].gridTransform = CastOpenVDBTransformToTransform(GetTranslatedFrameTransform(
-                    originGrid->constTransform(), openvdb::math::Vec3d(aabb.minX, aabb.minY, aabb.minZ) * ZIB_BLOCK_SIZE));
+                const auto& aabb = channelAABB[channelIndex];
+                sparseFrame.perChannelGridInfo[channelIndex].gridTransform = CastOpenVDBTransformToTransform(
+                    GetTranslatedFrameTransform(gridTransform, openvdb::math::Vec3d(aabb.minX, aabb.minY, aabb.minZ) * ZIB_BLOCK_SIZE));
             }
         }
 
         // Translate aabb to positive quarter of coordinate system.
-        aabb.maxX -= aabb.minX;
-        aabb.maxY -= aabb.minY;
-        aabb.maxZ -= aabb.minZ;
-        aabb.minX = 0;
-        aabb.minY = 0;
-        aabb.minZ = 0;
-        sparseFrame.boundingBox = aabb;
+        sparseFrame.boundingBox = {
+            0, 0, 0, std::numeric_limits<int32_t>::min(), std::numeric_limits<int32_t>::min(), std::numeric_limits<int32_t>::min()};
```

Comment:
We don't need `std::numeric_limits<int32_t>::min()`, since values are unsigned and we only use last 3 elemnts, we can just initialize everything with zeroes
```suggestion
        sparseFrame.boundingBox = {};
```

Reply:
Done